docs/api.md

```diff
@@ -3861,6 +3862,9 @@ Emitted when new service worker is created in the context.
 - `page` <[Page]> Page to create new session for.
 - returns: <[Promise]<[CDPSession]>> Promise that resolves to the newly created session.
 
+#### chromiumBrowserContext.serviceWorkers()
+- returns: <[Array]<[Worker]>> All existing service workers in the context.
```

Comment:
Do we have origin on Worker? Determining who owns a service worker via its script url is unfortunate.

Reply:
We don't have it at the moment. We could expose similar Page.serviceWorkers(), or add ServiceWorker class that would allow to get a list of its clients.